test/large_integration_tests/_expected_libclang_bindings.dart

```diff
@@ -7462,6 +7524,9 @@ typedef CXCursorVisitor = ffi.Pointer<
 /// various callbacks and visitors.
 typedef CXClientData = ffi.Pointer<ffi.Void>;
 
+/// Visitor invoked for each cursor found by a traversal.
+typedef CXCursorVisitorBlock = ffi.Pointer<ffi.Opaque>;
```

Comment:
Interesting. Looks like libclang will parse this and return a block pointer, even when objective c mode is not turned on.

https://intel.github.io/llvm-docs/clang_doxygen/Index_8h_source.html

Reply:
Seems like it only does this on mac. I'll make the objective C stuff in extractor.dart conditional on the language flag.